src/components/common/AccountSwitch.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { createFragmentContainer } from 'react-relay';
+import { graphql } from 'babel-plugin-relay/macro';
+
+import { WithStyles } from '@mui/styles';
+import Menu from '@mui/material/Menu';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import withStyles from '@mui/styles/withStyles';
+import createStyles from '@mui/styles/createStyles';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+import { navigateHelper } from '../../utils/navigateHelper';
+
+import { AppBreadcrumbs_viewer } from './__generated__/AppBreadcrumbs_viewer.graphql';
+
+const styles = theme => createStyles({});
+const styled = withStyles(styles);
+
+interface AccountSwitchProps extends WithStyles<typeof styles> {
+  viewer: AppBreadcrumbs_viewer;
+}
+
+const AccountsSwitch = styled(({ viewer, classes }: AccountSwitchProps) => {
+  const navigate = useNavigate();
```

Comment:
Also you can probably do 

```suggestion
  if (viewer.relatedOwners && viewer.relatedOwners.length <= 1) return null;
  const navigate = useNavigate();
```

Reply:
I think this is not resolved. We either need this or the same logic under `Stack`, don't we?

Reply:
I removed this logic under `Stack` and moved it to `AcoountSwitch`